backend/wmg/data/schemas/cube_schema_default.py

```diff
@@ -27,27 +29,25 @@
 # The full set of logical cube dimensions by which the cube can be queried.
 expression_summary_logical_dims = expression_summary_indexed_dims + expression_summary_non_indexed_dims
 
-filters = [tiledb.ZstdFilter(level=+22)]
-
 expression_summary_domain = tiledb.Domain(
     [
-        tiledb.Dim(name=cube_indexed_dim, domain=None, tile=None, dtype="ascii", filters=filters)
+        tiledb.Dim(name=cube_indexed_dim, domain=None, tile=None, dtype="ascii", filters=filters_categorical)
         for cube_indexed_dim in expression_summary_indexed_dims
     ]
 )
 
 # The cube attributes that comprise the core data stored within the cube.
 expression_summary_logical_attrs = [
-    tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters),  # TODO: Why uint64?
-    tiledb.Attr(name="sum", dtype=np.float32, filters=filters),
-    tiledb.Attr(name="sqsum", dtype=np.float32, filters=filters),
+    tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters_numeric),  # TODO: Why uint64?
```

Comment:
`Why uint64?` is a super-old comment. Would uint32 suffice? Since this is a cube, the attr size is probably irrelevant. If irrelevant, it's a good time to remove the comment.